auth/client/src/main/java/org/wildfly/security/auth/client/ElytronMessages.java

```diff
@@ -165,4 +165,7 @@ ConfigXMLParseException xmlUnableToIdentifyProvider(@Param Location location, St
     @Message(id = 9529, value = "Unsupported algorithm \"%s\" for %s type")
     ConfigXMLParseException xmlUnsupportedAlgorithmForType(@Param Location location, String algorithm, String type);
 
+    @Message(id = 14000, value = "At least one of the '%s' and '%s' cipher-suite attributes must be provided")
+    ConfigXMLParseException atLeastOneCipherSuiteAttributeMustBeProvided(String attribute1, String attribute2);
```

Comment:
I think this class is missing the valid ranges annotation at the top, can you please add that back in and add the new allocated range?

Reply:
Added.